### PR TITLE
renaming of the .sad file to .run

### PR DESCRIPTION
**Pull request by mandresm**
Changes the `.sad` extension by `.run`.

### CODE REVIEW

**Review by seb-wahl — CHANGES_REQUESTED** [review]
You should remove the .batch_system.py.swp (vim swap file) from the commit

**Comment by mandresm** [comment]
Done, and thanks!

**Review by denizural — APPROVED** [review]
sad file is gone, now I am very sad 😭 . Cold jokes aside, it looks fine. I assume that you grepped all sads in the code and the docs. Approve from my side. 👏 👏

**Comment by denizural** [comment]
Btw, if you use neovim, it will create the `.swp` file in another directory and won't mess up your repo.

File: src/esm_runscripts/backup/sadfile.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-class sadfile:
+class runfile:
```
Comment (pgierz) [inline src/esm_runscripts/backup/sadfile.py:1]: Not for now, but we should stick to common standards for class names. Normally they are capitalised. One more for the cleanup list.

**Review by pgierz — APPROVED** [review]

File: src/esm_runscripts/backup/sadfile.py (hunk 1 above)
Comment (denizural) [reply src/esm_runscripts/backup/sadfile.py:1]: Ah, I missed that. Here is a good read from PEP
https://www.python.org/dev/peps/pep-0008/#class-names

CamelCase is the naming convention for the classes. I remember that some classes in `esm_master` also violate that. Not too bad but the style police might be angry.

**Pull request merged**